System.Compiler/Nodes.cs

```diff
@@ -5609,6 +5609,19 @@ public Module(TypeNodeProvider provider, TypeNodeListProvider listProvider, Cust
       get{return this.metadataImportErrors;}
       set{this.metadataImportErrors = value;}
     }
+      
+    private ArrayList metadataImportWarnings;
+    /// <summary>
+    /// Holds information about non-critical errors.
+    /// For instance, lack of /names stream in the PDB is not a critical error and should be handled
+    /// gracefully.
+    /// </summary>
+    public ArrayList MetadataImportWarnings {
```

Comment:
:bulb: Seems like this could be the following:

```csharp
public List<Exception> MetadataImportWarnings
{
  get;
  set;
}
```

And the backing field `metadataImportWarnings` removed.